lib/simpler/router/route.rb

```diff
@@ -2,19 +2,34 @@ module Simpler
   class Router
     class Route
 
-      attr_reader :controller, :action
+      attr_reader :controller, :action, :params
 
       def initialize(method, path, controller, action)
         @method = method
         @path = path
         @controller = controller
         @action = action
+        @params = []
       end
 
       def match?(method, path)
-        @method == method && path.match(@path)
+        original_arr = @path.split('/').drop(1)
+        given_arr = path.split('/').drop(1)
```

Comment:
имя лучше поменять и при удалении элементов правильней быть более специфичным
ты же хочешь удалить именно пустые элементы
```suggestion
        request_path_elements = path.split('/').delete("")
```